.github/workflows/ci.yml

```diff
@@ -14,6 +14,9 @@ on:
     branches: ['**', '!update/**', '!pr/**']
     tags: [v*]
 
+permissions:
+  id-token: write
```

Comment:
I'm afraid this will bork everything 😅 maybe we can be lazy and just set this to read-write...

> If you specify the access for any of these scopes, all of those that are not specified are set to `none`.

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview

Reply:
updated